test/e2e/e2e_docker_helper.go

```diff
@@ -17,8 +17,7 @@ import (
 	"github.com/docker/docker/api/types/network"
 	"github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/archive"
-	"github.com/docker/docker/pkg/system"
-	. "github.com/onsi/gomega" // nolint: stylecheck
```

Comment:
Removing `gomega` is causing all sorts of lint and test errors I believe.

Reply:
The no lint comment was used for gomega but that's an unused import and vscode itself removes it while saving the file.